birdhouse/components/stac/config/proxy/conf.extra-service.d/stac.conf.template

```diff
@@ -9,7 +9,8 @@
         proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
         proxy_set_header X-Forwarded-Host $host:$server_port;
         proxy_buffering off;
-        include /etc/nginx/conf.d/cors.include;
+        proxy_hide_header 'Access-Control-Allow-Origin';
+        add_header Access-Control-Allow-Origin $stac_origin_allowed;
```

Comment:
I think the `cors.include` should still be there since it adds other definitions (allowed headers, allowed methods, etc.) which are also required.

I think `proxy_hide_header 'Access-Control-Allow-Origin';` will remove it.
Does `add_header` automatically reapply? I was under the impression explicit `proxy_pass_header` was needed to undo the hide. 

On top of this header, an `add_header Vary Origin;` would be required to advertise (as required by the standard) that it is Origin-dependant.